src/api.rs

```diff
@@ -0,0 +1,47 @@
+use reqwest;
+
+struct BulkUpdate {
+    collection: String,
+    auth_code: String,
+    current_hash: String,
+    new_hash: String,
+    repo: String,
+    files: Vec<File>,
+}
```

Comment:
Move this file to a new crate, `ft-api`. Keep this crate in this repo for now. When we are done we will move it to separate repo.